src/pages/katalog.tsx

```diff
@@ -1,11 +1,94 @@
+    "fa-book-open", "fa-shopping-bag", "fa-camera", "fa-phone-alt", "fa-ad"]
+
+    const icons = [iconsBase, iconsSponsor, iconsHunter, iconsPremium]
+
+    const packetColor1 = ["border-[#E2B7C9]", "border-[#D5759C]", "border-cerise", "border-yellow"]
+    const packetColor2 = ["bg-[#E2B7C9]", "bg-[#D5759C]", "bg-cerise", "bg-yellow"]
+
+    const stateAction = Array.from({ length: packets.length}, () => useState(false));
 
     return (
-        <div>
-            <p>Katalog</p>
+        <div className="pt-[200px] pb-[300px]">
+            <h1 className="uppercase text-cerise text-5xl font-medium text-center px-[10px] break-words">{t.catalog.header}</h1>
+
+            <div className="flex flex-row justify-between px-[10px] md:px-[80px] mt-[150px] overflow-x-auto overflow-y-hidden pb-6">
+                {packets.map((packet, i) => <div className="flex flex-col px-[20px] md:px-[10px]">
+                    <div className={`
+                    justify-between relative z-40 flex flex-col py-[30px] px-[px] h-[230px] w-[300px] lg:h-[225px] 2xl:h-[250px] md:w-[150px] lg:w-[200px] xl:w-[250px] 2xl:w-[300px] 
+                    ${packetColor1[i]} border-[3px] rounded-xl bg-slate-50 bg-opacity-10 items-center
+                    `}>
+                        <h2 className="text-white text-center text-3xl md:text-sm lg:text-xl xl:text-2xl 2xl:text-3xl">{t.catalog.packetType[i]}</h2>
+                        <div>
+                        <h2 className="text-yellow text-center text-sm lg:text-xl ">{t.catalog.info[i]}</h2>
+                        <h2 className="text-white text-center text-3xl lg:text-4xl pb-1 px-[10px]">{prices[i]}</h2>
+                        </div>
+                        <button onClick={() => stateAction[i][1](!stateAction[i][0])} className={` hover:scale-105 transition-transform hover:brightness-110 absolute h-[40px] w-[100px] ${packetColor2[i]} rounded-[40px] border-cerise bottom-0 translate-y-1/2`} 
+                        style={{ textShadow: '1px 1px 1px black, 1px -1px 1px black, -1px 1px 1px black, -1px -1px 1px black' }}>
+                            <i className={`${stateAction[i][0] ? "" : "rotate-180"} duration-200 fas fa-chevron-up text-white`}></i>
+                        </button>
+                    </div>
+                    <div className={`
+                    ${stateAction[i][0] ? " max-h-full py-[40px]"
+                              : "max-h-0 text-[0px] border-b-0"}
+                    md:w-[150px] lg:w-[200px] xl:w-[250px] 2xl:w-[300px] 
+                    px-[0px] duration-200 z-0 flex flex-col ${packetColor1[i]} border-[3px] border-t-0 rounded-b-xl bg-opacity-10 text-center bg-slate-50 `}>
+                    {packet.map((row, j) => (
+                            <div className={`flex flex-col py-2 items-center px-[50px] mt-10`}>
+                            <i className={`${stateAction[i][0] ? "text-2xl" : "text-[0px]"} fas ${icons[i][j]} duration-200 text-white`} 
```

Comment:
This transition looks quite weird to me. I think just transitioning on the height and setting overflow-y to hidden would look better. I used the trick in [this video](https://youtu.be/B_n4YONte5A) in the navbar to do that.